jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/FieldScope.java

```diff
@@ -112,6 +113,14 @@ public MethodScope findGetter() {
      * @return public getter from within the field's declaring class
      */
     private MethodScope doFindGetter() {
+        if (this.getMember().getType().isInstanceOf(Supplier.class)) {
+            ResolvedMethod[] methods = getContext().resolveWithMembers(this.getMember().getType()).getMemberMethods();
+            return Stream.of(methods)
+                    .filter(method -> method.getName().equals("get"))
```

Comment:
**question**: Collecting all methods only to then filter by name seems excessive. Isn't there an easy way to just look-up the one method we're interested in?